src/content/guides/production.md

```diff
@@ -113,7 +113,7 @@ Note the use of `merge()` in the environment-specific configurations to easily i
 
 ## NPM Scripts
 
-Now let's repoint our `scripts` to the new configurations. We'll use the _development_ one for our `webpack-dev-server`, `npm start`, script and the _production_ one for our `npm run build` script:
+Now let's repoint our `scripts` to the new configurations. We'll use the _development_ one for our `npm start` script and the _production_ one for our `npm run build` script:
```

Comment:
```suggestion
Now let's point our `scripts` to the new configuration files. `npm start` script, which uses `webpack-dev-server`, at _webpack.dev.js_ and `npm run build` script at _webpack.prod.js_:
```

Reply:
I think this sentence can be further improved.

What about this:

Now, let's modify our npm scripts to use the new configuration files. For the `start` script, which runs `webpack-dev-server`, we will use `webpack.dev.js`, and for the `build` script, which runs `webpack` to create a production build, we will use `webpack.prod.js`.

?

Reply:
amazing, i like it! go ahead

Reply:
Done!